### DISCUSSION TITLE
Minify JS files

### DISCUSSION THREAD

**Discussion by m-phil** [LiveHelperChat/livehelperchat#2131] [Ideas]
Hey there,
your JS files should be combined to one if possible and more important minified to reduce overhead and kB.
It would also be nice to add a aria-label and region or role to the <div id="lhc_status_container"> for better barrier-free accessibility.
Thx,
Phil

**Comment by remdex** [comment]
It already does that. Otherwise clarify your suggestion? and if multiple extensions are used after ready to go public just execute

```
php cron.php -s site_admin -c cron/util/generate_css -p 1 && gulp js-static
```

Check demo if in doubt - https://demo.livehelperchat.com/

Docs: https://doc.livehelperchat.com/docs/development/quick-guide
